editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -955,7 +948,6 @@ void SpriteFramesEditor::_animation_name_edited() {
 
 	EditorUndoRedoManager *undo_redo = EditorUndoRedoManager::get_singleton();
 	undo_redo->create_action(TTR("Rename Animation"), UndoRedo::MERGE_DISABLE, EditorNode::get_singleton()->get_edited_scene());
-	_rename_node_animation(undo_redo, false, edited_anim, "", "");
```

Comment:
I'm not sure what this line is for, but after removing it, the error "Animation '' doesn't exist." disappeared and I didn't notice any regressions.